packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -1,58 +1,46 @@
+// @ts-check
+
+// eslint-disable-next-line import/no-extraneous-dependencies
 import test from 'ava';
+import { Far } from '@agoric/marshal';
 
-import { makeAmountMath, MathKind } from '../../../src';
+import { amountMath as m, MathKind } from '../../../src';
+import { mockBrand } from './mockBrand';
 
 // The "unit tests" for MathHelpers actually make the calls through
 // AmountMath so that we can test that any duplication is handled
 // correctly.
 
-const mockBrand = harden({
-  isMyIssuer: () => false,
-  getAllegedName: () => 'mock',
-});
-
-const amountMath = makeAmountMath(mockBrand, MathKind.NAT);
-
-test('natMathHelpers', t => {
-  const {
-    getBrand,
-    getAmountMathKind,
-    make,
-    coerce,
-    getValue,
-    getEmpty,
-    isEmpty,
-    isGTE,
-    isEqual,
-    add,
-    subtract,
-  } = amountMath;
-
-  // getBrand
-  t.deepEqual(getBrand(), mockBrand, 'brand is brand');
-
-  // getAmountMathKind
-  t.deepEqual(getAmountMathKind(), MathKind.NAT, 'amountMathKind is nat');
-
-  // make
-  t.deepEqual(make(4), { brand: mockBrand, value: 4n });
+test('natMathHelpers make', t => {
+  t.deepEqual(m.make(4n, mockBrand), { brand: mockBrand, value: 4n });
```

Comment:
Should there be a test that `m.make(4, mockBrand)` produces the same result?  If not, then a test that non-bigInts cause a throw?

Reply:
Added! Numbers coerce to BigInts and are still accepted here.